Tests/IntegrationTests/IntegrationTests.Worker/CSharp/CSharpCodeTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace IntegrationTests.Worker.CSharp;
+
+using OJS.Workers.Common.Models;
+
+public class CSharpCodeTests : BaseStrategyTest<CSharpCodeSubmissionFactory, CSharpCodeParameters>, IClassFixture<RabbitMqAndWorkerFixture>
+{
+    public CSharpCodeTests(RabbitMqAndWorkerFixture fixture)
+        : base(fixture, new CSharpCodeSubmissionFactory()) { }
+
+    [Fact]
+    public async Task DotNet6_ShouldReturnCorrectAnswer_WhenCodeIsValid()
+    {
+        var submission = this.Factory.GetSubmission(new("Console.WriteLine(\"Hello SoftUni\");"));
+
+        var receivedSubmission = await this.Fixture.ProcessSubmission(submission);
+
+        // Assertions
```

Comment:
If we use comments to separate the AAA steps, let's put `// Arrange` and `// Act` also, or just remove the comments altogether, as this is relatively short test.